README.md

```diff
@@ -39,5 +41,5 @@ helm install istiod istio/istiod -n istio-system
 Run the plugin with Acorn:
 
 ```shell
-acorn run --name acorn-istio-plugin -i .
+acorn run --name acorn-istio-plugin .
```

Comment:
This is under the `##Development` section. Can you move the `acorn run --name acorn-istio-plugin -i .` command to be inside of `##Development` and have a seperate `##` weight tag for production runs (`acorn run ghcr.io/acorn-io/acorn-istio-plugin:prod` now that that's working 🥳 )?